Sources/ProjectDescription/Up/UpMint.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/// Up that installs Mint and packages.
+class UpMint: Up {
+    /// A Boolean value indicating whether installing the packages of the Mintfile globally.
+    let linkPackagesGlobally: Bool
+
+    /// Initializes the Mint up.
+    ///
+    /// - Parameter linkPackagesGlobally: A Boolean value indicating whether installing the packages of the Mintfile globally.
+    init(linkPackagesGlobally: Bool) {
+        self.linkPackagesGlobally = linkPackagesGlobally
+        super.init()
+    }
+
+    public enum CodingKeys: String, CodingKey {
+        case type
+        case linkPackagesGlobally
+    }
+
+    required init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        linkPackagesGlobally = try container.decode(Bool.self, forKey: .linkPackagesGlobally)
+        try super.init(from: decoder)
+    }
+
+    override func encode(to encoder: Encoder) throws {
```

Comment:
Since we are overriding the encoding and decoding logic, would you mind adding a test that ensures that there's parity between the encoding and decoding? You can use [this utility](https://github.com/tuist/tuist/blob/master/Sources/TuistSupportTesting/Extensions/XCTestCase+Extras.swift#L87) for that.